app/src/main/java/de/westnordost/streetcomplete/data/OsmModule.java

```diff
@@ -64,19 +64,29 @@ public static OsmConnection osmConnection(OAuthConsumer consumer)
 		return new OsmMapDataFactory();
 	}
 
+	public static int defaultOverpassTimeoutInMiliseconds(){
+		// see https://wiki.openstreetmap.org/wiki/Overpass_API/Overpass_QL#timeout:
+		// default value is 180 seconds
+		// give additional 4 seconds to get and process refusal from Overpass
+		// or maybe a bit late response rather than trigger timeout exception
+		return (180 + 4) * 1000;
+	}
+
```

Comment:
`private static int DEFAULT_OVERPASS_TIMEOUT_IN_SECONDS`?
The link to the Overpass Api documentation  note about the 4 seconds is probably enough here

Reply:
I moved to `OVERPASS_QUERY_TIMEOUT_IN_MILISECONDS` name as 184 seconds (including extra buffer) is not actually equal to overpass timeout. 

I used miliseconds to avoid later `* 1000`.

Hopefully it is OK.